core/arch/arm/dts/stm32mp13xf.dtsi

```diff
@@ -27,10 +27,11 @@
 	};
 
 	pka: pka@54006000 {
-		compatible = "st,stm32mp13-pka64";
+		compatible = "st,stm32mp13-pka";
 		reg = <0x54006000 0x2000>;
 		interrupts = <GIC_SPI 105 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&rcc PKA>;
+		clocks = <&rcc PKA>, <&rcc RNG1_K>;
```

Comment:
Looking at this, the secure state of the RNG clock is inherited from the secure state of the RNG peripheral, so I'm tempted to add `<&etzpc STM32MP1_ETZPC_RNG_ID>` in the access controllers property